docs/index.rst

```diff
@@ -0,0 +1,28 @@
+CATChem
+==============
+
+**CATChem**  (Configurable ATmospheric Chemistry) is a modelling 
+component that includes all chemical and aerosol processes needed to perform 
+atmospheric chemistry and composition simulations within a model through a 
+flexible, easy to modify, and well-documented infrastructure. CATChem will 
+include the following processes: chemical kinetics, aerosols, photolysis, 
+wet deposition, dry deposition, connections to emissions, and connection to 
+physics schemes. The first use of CATChem will be connecting it to NOAA's 
+Unified Forecasting System to create UFS-Chem. CATChem and UFS-Chem are 
```

Comment:
Could provide a link to UFS

Reply:
I linked https://ufscommunity.org/, is that what you had in mind?